PODMAN.md

```diff
@@ -13,5 +13,11 @@ This will use `fuse-overlayfs` as storage layer. If the performance is not
 satisfactory, consider running podman as root to use plain `overlayfs2`:
 
 ```
-sudo podman system service -t 0 unix:///${HOME}/podman.sock
+mkdir -p $XDG_RUNTIME_DIR/podman
+sudo podman system service -t 0 unix:///$XDG_RUNTIME_DIR/podman/podman.sock
+sudo chown $USER $XDG_RUNTIME_DIR/podman/podman.sock
```

Comment:
Ditto the above shouldn't be required.

Reply:
When running rootful podman the socket intially belongs to `root:root`, which the user could not access then.

I documented it this way so you can run rootful podman without make any other changes to the repository files (when #741 and https://github.com/kubevirt/kubevirt/pull/7160 are merged).

Reply:
Right but what you end up with there isn't rootless, it's rootful podman that's accessiable to your user via that socket.

I still think we should just document the working setup at the moment of using rootful podman until the pure rootless usecase works:

```
$ sudo systemctl start podman
$ sudo ln -sf /var/run/podman/podman.sock /root/podman.sock # Workaround pull/7160
$ sudo make cluster-up
```

Reply:
It was my intention to run a rootful podman accessible to the user via that socket. You can then run `make cluster-up` without `sudo` so that at least not everything is run as root. That's similar to how it's usually done with docker. [1](https://docs.docker.com/engine/install/linux-postinstall/#manage-docker-as-a-non-root-user) In the future it would also be compatible with the default rootless setup (no path changing).

Another possible way would be

```
sudo systemctl start podman
mkdir -p $XDG_RUNTIME_DIR/podman
sudo ln -sf /var/run/podman/podman.sock $XDG_RUNTIME_DIR/podman/podman.sock
sudo chown $USER /var/run/podman/podman.sock
```

Or maybe introduce a detection whether `make cluster-up` is run as root / non-root and select the appropriate socket automatically?

Reply:
ACK apologies that's entirely valid and better than just running everything under sudo as you say.